Cargo.toml

```diff
@@ -17,3 +16,11 @@ serde = { version = "*", features = ["derive"] }
 serde_derive = "*"
 termion = "1.5"
 tui = "0.10.0"
+
+[dependencies.confy]
+version = "0.4.0"
+features = ["yaml_conf"]
+default-features = false
+
+[patch.crates-io]
```

Comment:
Confy > 0.5.0 is required as toml has bad handling of objects in lists. The maintainers 'forgot' to push to crates.io, thus the patch